<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleFigure</class>
 <widget class="QWidget" name="SimpleFigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="dataPlot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>641</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Close">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Accept">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Reject">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>510</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_XAxisMin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Axis Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_XAxisMax">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Axis-Max:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_YAxisMin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Axis-Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_YAxisMax">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Axis-Max:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_XAxisMin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_YAxisMin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_XAxisMax">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>8192</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_YAxisMax">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_LogScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowLogScale</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
